graylog2-server/src/main/resources/web-interface/index.html.template

```diff
@@ -5,16 +5,15 @@
     <meta name="robots" content="noindex, nofollow">
     <meta charset="UTF-8">
     <title>${title}</title>
-    <base href="${baseUri}">
-    <link rel="shortcut icon" href="assets/favicon.png">
+    <link rel="shortcut icon" href="${appPrefix}assets/favicon.png">
```

Comment:
Why not use `${appPrefix}/assets/favicon.png` here (and in the other `script` tags)?
Then we could simply strip all trailing slashes from `http_external_uri`.

Reply:
I looked at the usages of `RestTools#buildExternalUri()` and saw that we expect the URI to have a trailing slash. That's why I decided to make sure a URI returned from `#buildExternalUri()` has a trailing slash and adjusted the template to remove the `/` to avoid duplicate ones.